cEP-0027.md

```diff
@@ -0,0 +1,155 @@
+# Improvise coala Testing API
```

Comment:
I don't think "Improvise" is the right word. Do you mean "improve"? 

I would just go with "Bears testing API" or something. Your work will be specifically on the bear lib, so we can be this specific.